client/i18n.js

```diff
@@ -0,0 +1,37 @@
+import i18n from 'i18next';
+import { initReactI18next } from 'react-i18next';
+// import detector from 'i18next-browser-languagedetector';//
+// import Fetch from 'i18next-fetch-backend';
+// import Fetch from 'i18next-fetch-backend';
+import Backend from 'i18next-http-backend';
+
+const fallbackLng = ['en'];
+const availableLanguages = ['en', 'es'];
+
+const options = {
+  loadPath: '/locales/{{lng}}/translations.json',
+  requestOptions: { // used for fetch, can also be a function (payload) => ({ method: 'GET' })
+    mode: 'no-cors'
+  },
+  allowMultiLoading: false, // set loadPath: '/locales/resources.json?lng={{lng}}&ns={{ns}}' to adapt to multiLoading
+};
+
+i18n
+  .use(initReactI18next) // pass the i18n instance to react-i18next.
+  .use(Backend).init({
+    lng: 'en',
+    defaultNS: 'menu',
+    fallbackLng, // if user computer language is not on the list of available languages, than we will be using the fallback language specified earlier
+    debug: true,
+    backend: options,
+    getAsync: false,
+    initImmediate: false,
+    useSuspense: true,
+    whitelist: availableLanguages,
+    interpolation: {
+      escapeValue: false
+    },
+    saveMissing: true,
```

Comment:
`Menu.Open` is missing, but I'm not sure what `saveMissing` actually does. Would be great if it logged it somewhere!

<img width="339" alt="Screenshot 2020-06-29 18 07 53" src="https://user-images.githubusercontent.com/1762/86029491-85da5c00-ba33-11ea-9164-df1e5a477299.png">